bundle/trampoline/python_wheel.go

```diff
@@ -80,6 +80,7 @@ func (transformWheelTask) Apply(ctx context.Context, b *bundle.Bundle) diag.Diag
 		return nil
 	}
 
+	b.Metrics.AddSetField("python_wheel_wrapper")
```

Comment:
Can we use the full bundle configuration path to avoid ambiguity and always be consistent?
```suggestion
	b.Metrics.AddSetField("experimental.python_wheel_wrapper")
```

Reply:
Practically it doesn't make any sense to set python_wheel_wrapper to false, generally we need this metric to see if this property is even used, whatever the value is

Reply:
The problem is documentation for set_fields in the protobuf / code states it tracks whether the field is set. Misusing it here will lead to inconsistent data in our tables.

I think we should be consistent and either use set_fields to indicate `experimental.python_wheel_wrapper` is set regardless of value OR use the bool_values entry to store arbitary bool kv pairs.

Reply:
Changed to use bools